### PR TITLE
fix(TBD-12992): Get error 'org.apache.spark.streaming.pubsub cannot be resolved to a type' when using tPubsubOutput component

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.spl30x/plugin.xml [hunk 1]
```diff
@@ -817,7 +821,14 @@
             <group id="SPARK-BATCH-GROUP-SPARK_3_0_x"/>
             <library id="py4j-0.10.9.1.jar"/>
         </libraryNeededGroup>
-
+        
+        <!-- Spark Streaming PubSub group -->
+        <!-- Since no spark-streaming-pubsub lib for spark 3.x  spark-streaming-pubsub_2.12-2.4.0.jar will be used as workaround-->
+        <libraryNeededGroup description="Spark streaming PubSub libraries for spl30x" id="PUBSUB-GROUP-SPARK_3_0_x" name="PUBSUB-GROUP-SPARK_3_0_x">
+            <library id="spark-streaming-pubsub_2.12-2.4.0.jar"/>
+            <group id="SPARK-STREAMING-GROUP-SPARK_3_0_x"/>
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.spl30x/plugin.xml:829]: I don't think this line is needed. Every Spark streaming job should already have SPARK-STREAMING-GROUP-SPARK_3_0_x imported.

Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.spl30x/plugin.xml:829]: Good point. Thanks. Will change that.

**Review by cbadillo1603 — APPROVED** [review]
reviewed

**Review by lbourgeois — APPROVED** [review]
LGTM

**Pull request merged**